### DISCUSSION TITLE
bus error after install

### DISCUSSION THREAD

**Discussion by epi052** [epi052/feroxbuster#555] [Q&A]
> I'm getting"bus error" after installation
> https://github.com/epi052/feroxbuster/discussions/149#discussioncomment-2650639

hey there @0xVenus

I created a new discussion based off your initial post, so it would be easier to track.  

Can you let me know more about your problem? 
- feroxbuster version
- operating system
- exact error messge you're seeing
- how you installed feroxbuster
- anything else you can think of that might assist in troubleshooting the problem

thanks!

**Comment by 0xVenus** [comment]
**Feroxbuster latest version**
Ubuntu focal fossa
Getting bus error anytime I want to run the program
I installed via the command **curl -sL https://raw.githubusercontent.com/epi052/feroxbuster/master/install-nix.sh | bash**

**Reply by epi052** [reply]
could you copy and paste the exact error here when you get a chance?

also, could you please run the following command, where `/your/path/to/feroxbuster` is the location to which you downloaded the binary via curl? 
```
ls -l /your/path/to/feroxbuster
```
